nio/client/async_client.py

```diff
@@ -1233,6 +1235,34 @@ async def devices(self) -> Union[DevicesResponse, DevicesError]:
 
         return await self._send(DevicesResponse, method, path)
 
+    @logged_in
+    async def update_device(
+            self,
+            device_id: str,
+            content: Dict[str, str]
+    ) -> Union[UpdateDeviceResponse, UpdateDeviceError]:
+        """Update the metadata of the given device.
+
+        Returns either a `UpdateDeviceResponse` if the request was successful or
+        a `UpdateDeviceError` if there was an error with the request.
+
+        Args:
+            device_id (str): The device for which the metadata will be updated.
+            content (Dict): A dictionary of metadata values that will be
```

Comment:
You'll probably want to describe in the docstring what the dict maps from and to (str and str), either in the type hint or just in the text

Reply:
Done